pandas/__init__.py

```diff
@@ -264,13 +275,39 @@ def __getattr__(self, item):
                 FutureWarning,
                 stacklevel=2,
             )
+
             try:
                 return getattr(self.np, item)
             except AttributeError:
                 raise AttributeError(f"module numpy has no attribute {item}")
 
     np = __numpy()
 
+    class __Datetime:
```

Comment:
Why two underscores instead of one?

Reply:
This was just to make it  consistent with how __numpy() was being used - same for __get__attr__ as thats how its used in `pandas/tests/api/test_api.py::TestPDApi::test_depr`
but i can revert if still needed